x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.test.ts

```diff
@@ -1249,6 +1251,15 @@ describe('sources routes', () => {
   });
 
   describe('GET /api/workplace_search/sources/create', () => {
+    const tokenPackage = 'sEkReTZ';
```

Comment:
That's fair. I've made it more explicitly a placeholder.

Reply:
I will now I couldn't stop seeing spongebob mocking meme whenever reading I was reading `sEkReTZ` 🙈

Reply:
OH WAIT. spongebob mocking... test mocking... I think I just got this incredible inception-level joke. Sadly this is too much for my tiny monkey brain James, I can't be distracted like this or I go off and start browsing memes instead of working :trollface: